### PR TITLE
our own eliotutil 

### PR DESCRIPTION
**Pull request by meejah**
...with former fields from tahoe

### CODE REVIEW

**Comment by vu3rdd** [comment]
This goes further. However, it fails because tests invokes `tahoe magic-folder`.

**Comment by vu3rdd** [comment]
It works fine with tahoe-lafs 1.14.0.

File: p [hunk 1]
```diff
@@ -0,0 +1,87 @@
+diff --git a/integration/test_general_cli.py b/integration/test_general_cli.py
```
Comment (exarkun) [inline p:1]: What is this whole file?

File: src/magic_folder/magic_folder.py [hunk 2]
```diff
@@ -38,6 +38,11 @@
     yamlutil,
     eliotutil,
 )
+from util.eliotutil import (
```
Comment (exarkun) [inline src/magic_folder/magic_folder.py:41]: Let's go with explicit relative imports ... `from .util.eliotutil import ...`

File: p (hunk 1 above)
Comment (vu3rdd) [reply p:1]: hmm.. I have no idea what that is and how that sneaked in. @meejah do you know?

Comment (meejah) [reply p]: oh, weird that's from another PR of mine I think

**Pull request merged**